av/container/core.pyx

```diff
@@ -201,6 +203,7 @@ def open(file, mode=None, format=None, options=None,
         reading on Python 2 (returning ``str`` instead of ``unicode``).
     :param str metadata_errors: Specifies how to handle encoding errors; behaves like
         ``str.encode`` parameter. Defaults to strict.
+    :param int bufsize: Decoder buffer size. Defaults to 32k.
```

Comment:
I think a better description is needed for the parameter, if it is used both for input and output streams I'm guessing "Decoder buffer size" isn't quite accurate? Also it would be nice to state what unit it is in.. bytes presumably?

Reply:
Thanks, I'll expand the description.

`buffer_size` sounds good, because it matches the name of the corresponding `avio_alloc_context` parameter.